docs/sql/sql-full-text.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: default
+title: Full-Text Search
+parent: SQL
+nav_order: 2
+---
+
+# Full-text search
+
+Use SQL commands for full-text search. For more information about full-text queries in Elasticsearch, see [Full-text queries](../../elasticsearch/full-text/).
+
+
+
+---
+
+#### Table of contents
+1. TOC
+{:toc}
+
+
+---
+
+## Match
+
+To search for text in a single field, use `MATCHQUERY` or `MATCH_QUERY` functions.
+
+Pass in your search query and the field name that you want to search against.
+
+
+```sql
+SELECT account_number, address
+FROM accounts
+WHERE MATCH_QUERY(address, 'Holmes')
+```
+
+Or,
```

Comment:
Let's be a little more formal here. "Alternate syntax:"

Reply:
Done.